### ISSUE TITLE
AttributeError: 'ListPortInfo' object has no attribute 'subsystem'

### ISSUE THREAD

**Issue by wwj718** [virtualabs/btlejack#8]
```
> btlejack -s
BtleJack version 1.1

Traceback (most recent call last):
  File "/usr/local/bin/btlejack", line 11, in <module>
    load_entry_point('btlejack==1.1.1', 'console_scripts', 'btlejack')()
  File "/usr/local/lib/python3.6/site-packages/btlejack/__init__.py", line 228, in main
    supervisor = CLIAccessAddressSniffer(verbose=args.verbose)
  File "/usr/local/lib/python3.6/site-packages/btlejack/ui.py", line 326, in __init__
    super().__init__()
  File "/usr/local/lib/python3.6/site-packages/btlejack/supervisors.py", line 86, in __init__
    self.interface = SingleSnifferInterface()
  File "/usr/local/lib/python3.6/site-packages/btlejack/jobs.py", line 24, in __init__
    self.link = Link(interface=device, baudrate=115200)
  File "/usr/local/lib/python3.6/site-packages/btlejack/link.py", line 42, in __init__
    if port.subsystem == 'usb':
AttributeError: 'ListPortInfo' object has no attribute 'subsystem'
```

macOS 10.13.5

**Comment by binloan** [comment]
Having the same issue. `ListPortInfo` seems to not work for MacOS.

**Comment by virtualabs** [comment]
I don't have a Mac so it would be difficult for me to test this one. It seems that 'subsystem' is filled only in Linux and does not exist when running on MacOS. Can someone try to comment line 42 and see if it solves the problem ?

**Comment by wwj718** [comment]
@virtualabs  try to comment line 42 

the output is 

```
➜  scratch_adapter git:(master) ✗ btlejack -s
BtleJack version 1.1

[!] Please connect a compatible Micro:Bit in order to use BtleJack
```

**Comment by virtualabs** [comment]
No other modification required ? Your previous post stated it was not ok.

**Comment by wwj718** [comment]
it is mistake.
it works well

**Comment by virtualabs** [comment]
Allright then, I will merge this fix :)

**Comment by virtualabs** [comment]
You're right. I fixed this one too. I'm tired at the moment, so I'll wait a few days before pushing a new release on both github and Pypi. Expect version 1.1.3 in a few days, or use master branch if you cannot wait.

**Comment by virtualabs** [comment]
I'm reopening this issue as it is still officially not fixed.

**Comment by virtualabs** [comment]
Fixed (well, normally) in v1.2.0.
